tests/core/test_eodata_io.py

```diff
@@ -520,17 +490,16 @@ def test_lazy_loading_plus_overwrite_patch(fs_loader, folder_name, eopatch, use_
         (FeatureIOTimestamps, [datetime.datetime(2017, 1, 1, 10, 4, 7), datetime.datetime(2017, 1, 4, 10, 14, 5)]),
     ],
 )
-@pytest.mark.parametrize("compress_level", [0, 1])
```

Comment:
why are you removing this parametrization here?

Reply:
my bad, reverted